python_modules/libraries/dagster-components/dagster_components/components/dbt_project/component.py

```diff
@@ -32,30 +32,15 @@
 from dagster_components.resolved.core_models import (
     AssetAttributesModel,
     AssetPostProcessor,
-    AssetPostProcessorModel,
     OpSpec,
-    OpSpecModel,
     ResolutionContext,
 )
-from dagster_components.resolved.metadata import ResolvableFieldInfo
-from dagster_components.resolved.model import ResolvableModel, ResolvedFrom, Resolver
+from dagster_components.resolved.model import Resolved, Resolver
 from dagster_components.scaffold import scaffold_with
 from dagster_components.utils import TranslatorResolvingInfo
 
 
-class DbtProjectModel(ResolvableModel):
-    dbt: DbtCliResource
-    op: Optional[OpSpecModel] = None
-    asset_attributes: Annotated[
-        Optional[Union[str, AssetAttributesModel]],
-        ResolvableFieldInfo(required_scope={"node"}),
-    ] = None
-    asset_post_processors: Optional[Sequence[AssetPostProcessorModel]] = None
-    select: str = "fqn:*"
-    exclude: Optional[str] = None
-
-
-def resolve_translator(context: ResolutionContext, model: DbtProjectModel) -> DagsterDbtTranslator:
+def resolve_translator(context: ResolutionContext, model) -> DagsterDbtTranslator:
```

Comment:
how should we tell users to type `model`? It's a bit of a bummer that it's really just `Any`, but I get why it has to be that way.

not necessary for this set of changes, but maybe we could provide functions on the `Model` class? i.e.

```python
def resolve_translator(context, model: Model):
    attributes = model.get_field("attributes", AssetAttributesModel)
```

This way we can at least get some typing information into this function